sdk/sdk-core/aar/src/main/java/mil/emp3/core/mapgridlines/AbstractMapGridLine.java

```diff
@@ -320,6 +320,13 @@ protected IFeature createLabelFeature(IGeoPosition position, String text, String
             label.setAzimuth(azimuth);
         }
 
+        if (this.currentCamera.getRoll() != 0.0) {
+            double azimuth = label.getAzimuth() + this.currentCamera.getRoll();
+
+            azimuth = ((((azimuth + 180) % 360.0) + 360.0) % 360.0) - 180.0;
+            label.setAzimuth(azimuth);
+        }
+
```

Comment:
1. There is a separate check just above this for Heading. If both are non-zero does this work? Would it be a good idea to add this normalization expression to global where all can use?